core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2193,11 +2193,13 @@ private[spark] class DAGScheduler(
    * Return true when:
    *  1. Waiting for decommission start
    *  2. Under decommission process
-   * Return false when:
-   *  1. Stopped or terminated after finishing decommission
-   *  2. Under decommission process, then removed by driver with other reasons
+   *  3. Stopped or terminated after finishing decommission
+   *  4. Under decommission process, then removed by driver with other reasons
+   * Return false in case 3 and 4 when removed executors info are not retained.
+   * The max size of removed executors is controlled by
+   * spark.scheduler.maxRetainedRemovedExecutors
    */
-  private[scheduler] def isExecutorDecommissioningOrDecommissioned(
+  private def isExecutorDecommissioningOrDecommissioned(
```

Comment:
Shall we revert this change because this PR doesn't use `isExecutorDecommissioningOrDecommissioned`?